### DISCUSSION TITLE
Disabled workflows can still be called from other workflows

### DISCUSSION THREAD

**Discussion by blujupiter32** [community/community#60389] [Actions]
### Select Topic Area

Bug

### Body

[Calling a reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow) does not check whether the called workflow has been [disabled](https://docs.github.com/en/actions/using-workflows/disabling-and-enabling-a-workflow#disabling-a-workflow), so a disabled workflow can still be run if another workflow calls it via this feature. To fix this, I propose that jobs which call disabled workflows should be skipped.

I've created a minimal example [repository](https://github.com/blujupiter32/reusable-workflows-bug-60389) with the behaviour shown in a [workflow run](https://github.com/blujupiter32/reusable-workflows-bug-60389/actions/runs/5494347102).

**Comment by zachspar** [comment]
This seems to be an [intended feature](https://docs.github.com/en/actions/using-workflows/reusing-workflows#overview) and not a bug.
> If you reuse a workflow from a different repository, any actions in the called workflow run as if they were part of the caller workflow.

If you don't want other repos to use it, then you can [tweak the settings](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#managing-github-actions-permissions-for-your-repository).
